docker-compose-base.yml

```diff
@@ -398,6 +398,10 @@ services:
       ZOOKEEPER_PW: ${ZK_ROOT_PW}
       KAFKA_SECRET: ${KAFKA_SECRET}
 
+      ELASTICSEARCH_HOST: http://elasticsearch:9200
+      ELASTICSEARCH_USER: admin
+      ELASTICSEARCH_PW: ${ELASTIC_PASSWORD}
```

Comment:
Can we try and be consistent on naming of env vars? Some examples:
- `ES_CONSUMER_CONFIG_PATH`
- `ELASTICSEARCH_HOST`
- `ELASTICSEARCH_PW`
- `ELASTIC_PASSWORD`

Is it possible to pick a prefix format and stick with it (ELASTIC/ELASTICSEARCH/ES), likewise whether to call it PASSOWRD or PW?

Reply:
:+1: